docs/USAGE-GUIDE.md

```diff
@@ -239,63 +238,62 @@ s2n-tls usually prefers AES over ChaCha20. However, some clients-- particularly
-|   "20160824"   |      X       |     X    |      X        |          |
-|   "20160804"   |      X       |     X    |      X        |          |
-|   "20160411"   |      X       |     X    |      X        |          |
-|   "20150306"   |      X       |     X    |      X        |          |
-|   "20150214"   |      X       |     X    |      X        |          |
-|   "20150202"   |      X       |     X    |      X        |          |
-|   "20141001"   |      X       |     X    |      X        |          |
-|   "20140601"   |      X       |     X    |      X        |          |
-|   "20190120"   |      X       |     X    |      X        |          |
-|   "20190121"   |      X       |     X    |      X        |          |
-|   "20190122"   |      X       |     X    |      X        |          |
-|   "20190801"   |      X       |     X    |      X        |    X     |
-|   "20190802"   |      X       |     X    |      X        |    X     |
-|   "20200207"   |      X       |     X    |      X        |    X     |
-|   "20230317"   |      X       |     X    |               |    X     |
-|   "rfc9151"    |      X       |     X    |               |    X     |
+|    version    | RSA PKCS1 | ECDSA | SHA-1 Legacy | RSA PSS |
+|---------------|-----------|-------|--------------|---------|
+|    default    |     X     |       |       X      |         |
+| default_tls13 |     X     |   X   |       X      |    X    |
+| default_fips  |     X     |   X   |              |         |
+|   20190214    |     X     |   X   |       X      |         |
+|   20170718    |     X     |       |       X      |         |
+|   20170405    |     X     |       |       X      |         |
+|   20170328    |     X     |       |       X      |         |
+|   20170210    |     X     |       |       X      |         |
+|   20160824    |     X     |       |       X      |         |
+|   20160804    |     X     |       |       X      |         |
+|   20160411    |     X     |       |       X      |         |
+|   20150306    |     X     |       |       X      |         |
+|   20150214    |     X     |       |       X      |         |
+|   20150202    |     X     |       |       X      |         |
+|   20141001    |     X     |       |       X      |         |
+|   20190120    |     X     |       |       X      |         |
+|   20190121    |     X     |       |       X      |         |
+|   20190122    |     X     |       |       X      |         |
+|   20190801    |     X     |   X   |       X      |    X    |
+|   20190802    |     X     |   X   |       X      |    X    |
+|   20200207    |           |   X   |              |    X    |
+|   20230317    |     X     |   X   |              |    X    |
+|    rfc9151    |     X     |   X   |              |    X    |
 
 Note that the default_tls13 security policy will never support legacy SHA-1 algorithms in TLS1.3, but will support
 legacy SHA-1 algorithms in CertificateVerify messages if TLS1.2 has been negotiated.
```

Comment:
While you're here, maybe this should be modified. This statement is true for any TLS1.3 policy, not just default_tls13.